cmd/openshift-newrelic-synthetics/sync/command.go

```diff
@@ -84,19 +90,19 @@ func syncSynthetics(client *newrelic.NewRelic, routes []routev1.Route, location
 
 		tags[m.Name] = []entities.Tag{
 			{
-				Key: entityutils.TagOpenShiftRouteNamespace,
+				Key:    entityutils.TagOpenShiftRouteNamespace,
```

Comment:
are the extra spaces intentional?

Reply:
Yeah... go fmt enforces them... no choice on my part.